src/apps/testapps/testH3CellArea.c

```diff
@@ -39,10 +39,22 @@ SUITE(h3CellArea) {
         for (int res = 0; res <= MAX_H3_RES - 1; res++) {
             H3Index cell;
             t_assertSuccess(H3_EXPORT(latLngToCell)(&gc, res, &cell));
-            double area = H3_EXPORT(cellAreaKm2)(cell);
+            double area;
+            t_assertSuccess(H3_EXPORT(cellAreaKm2)(cell, &area));
 
             t_assert(fabs(area - areasKm2[res]) < 1e-8,
                      "cell area should match expectation");
         }
     }
+
+    TEST(cell_area_invalid) {
+        H3Index invalid = 0xFFFFFFFFFFFFFFFF;
```

Comment:
Nit: Maybe explain why you don't need a more comprehensive list of bad indexes (i.e. covered elsewhere)?